src/py/_pyodide/_core_docs.py

```diff
@@ -223,7 +223,7 @@ def to_bytes(self) -> bytes:
         an ArrayBuffer view.
         """
 
-    def to_file(self, file: IOBase, /):
+    def to_file(self, file: IOBase, /) -> Any:
```

Comment:
```suggestion
    def to_file(self, file: IOBase, /) -> None:
```
https://github.com/pyodide/pyodide/blob/af972267c7969c6f023da709851463ec99e7e4ac/src/core/jsproxy.c#L1724